charts/rancher-monitoring/v0.0.4/charts/prometheus/templates/prometheus.yaml

```diff
@@ -80,6 +80,9 @@ spec:
   listenLocal: true
 {{- end }}
   podMetadata:
+    {{- if .Release.IsInstall }}
+    creationTimestamp: {{ now | date "2006-01-02T15:04:05Z" | quote }}
```

Comment:
what is this specific 2006 date for?

Reply:
This is the go date template format and the current date format is required for Prometheus schema validation.